README.md

```diff
@@ -317,6 +317,50 @@ export default withStyles(({ color, unit }) => ({
 
 `className` and `style` props must not be used on the same elements as `css()`.
 
+#### Example with React-Router's `Link`
```

Comment:
It would be nice to add `## Examples` and use a 3rd level heading here instead of a 4th level heading (e.g. `### With React Router's <Link>`)